Sources/SentryCrash/Recording/Tools/SentryCrashObjC.c

```diff
@@ -806,6 +806,9 @@ bool sentrycrashobjc_ivarNamed(const void* const classPtr, const char* name, Sen
     const struct ivar_list_t* ivars = getClassRO(classPtr)->ivars;
     uintptr_t ivarPtr = (uintptr_t)&ivars->first;
     const struct ivar_t* ivar = (void*)ivarPtr;
+    if (ivars == NULL) {
```

Comment:
Can we maybe add a comment why `ivars` can be `NULL` or why we need this change? Furthermore wouldn't it make sense to do this check right after `const struct ivar_list_t* ivars = getClassRO(classPtr)->ivars;`?

Reply:
this is a kstenerud/KSCrash fork and this makes tests flaky, the reason is unknown, but it should not throw NPE but rather return false if there's an invalid memory address.

I believe `ivars` loses its address after that line or it'd throw NPE already in the next line and this fix wouldn't fix the issue.

just commenting on it because I was debugging it with @bruno-garcia

Reply:
Okay great. Then let's add this info as a comment?